tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraServicesDiscoveryRoutesContract.scala

```diff
@@ -42,7 +42,7 @@ trait LinagoraServicesDiscoveryRoutesContract {
     assertThatJson(response)
       .isEqualTo(
         s"""{
-           |	"linShareApiUrl": "https://linshare.linagora.com/linshare/webservice",
+           |	"linshareApiUrl": "https://linshare.linagora.com/linshare/webservice",
```

Comment:
Maybe we need to notify the mobile team about the change here.

Reply:
No need really... you can put whatever as a key now.

Or maybe you prefer to make that change on the configuration sample file to avoid confusion?